runtime/lua/vim/treesitter/dev.lua

```diff
@@ -391,7 +395,7 @@ function M.inspect_tree(opts)
     buffer = buf,
     callback = function()
       if not api.nvim_buf_is_loaded(b) then
-        return true
+        return
```

Comment:
Why change this? `return true` will cause the autocommand to be removed, which sounds like what we want in this case?

Reply:
My bad. I had no idea that the return value of an autocommand's callback had that meaning (doesn't seem to be [documented](https://neovim.io/doc/user/lua-guide.html#lua-guide-autocommands)) and I thought they were mistakes. I'll undo the changes.

Reply:
It's documented in `nvim_create_autocmd`:

```
                 • callback (function|string) optional: Lua function (or
                   Vimscript function name, if string) called when the
                   event(s) is triggered. Lua callback can return true to
                   delete the autocommand, and receives a table argument with
                   these keys:
```

But I do admit it's not the most obvious or discoverable feature.

Reply:
The `return true` pattern was chosen to mimic `nvim_buf_attach`, which similarly lets callbacks use `return true` to detach.

Reply:
Oh I cannot read then. Thanks for the tip!